servers/src/main/java/tachyon/master/InodeFile.java

```diff
@@ -35,6 +37,12 @@
  */
 public class InodeFile extends Inode {
   /**
+   * The default permission for directory is 0755.
+   * The default permission for file is 0644.
+   */
+  private static final FsPermission SUMASK = new FsPermission((short) 0111);
```

Comment:
I think it later. I found this should not be configurable. Because we already have the configurable
key "tachyon.tfs.permission.umask". The 0111 umask will not changed. This design is the same as HDFS.